source/guide_grafana-loki.rst

```diff
@@ -0,0 +1,277 @@
+.. highlight:: console
+
+.. author:: Franz Wimmer <https://codefoundry.de>
+
+.. tag:: lang-go
+.. tag:: monitoring
+.. tag:: logs
+.. tag:: logshipper
+.. tag:: observability
+.. tag:: audience-admins
+
+.. sidebar:: About
+
+  .. image:: _static/images/loki.png
+      :align: center
+
+############
+Grafana Loki
+############
+
+.. tag_list::
+
+Loki is a log aggregation system designed to store and query logs from all your applications and infrastructure.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :lab:`Grafana <guide_grafana>`
+
+License
+=======
+
+Loki is licensed under the GNU Affero General Public License (AGPL).
+
+All relevant legal information can be found here
+
+  * https://github.com/grafana/loki/blob/main/LICENSE
+
+Prerequisites
```

Comment:
please state here as well, that we need a running Instance of Grafana to use this guide